onnxruntime/core/providers/cpu/tensor/transpose.cc

```diff
@@ -32,6 +32,29 @@ static inline void IncrementIndex(std::vector<int64_t>& index, const std::vector
   }
 }
 
+// Combines the two previous functions.
```

Comment:
Please include info in the comment on why we have extra code that combines/duplicates what two standalone functions do.

Reply:
Sorry, I removed the function because they were not needed anymore without updating this comment. Just did it.